activities/tool-management/nextcloud.md

```diff
@@ -0,0 +1,29 @@
+---
+type: resource
+---
+
+# Nextcloud
+
+Nextcloud is a suite of client-server software for creating and using file hosting services. Nextcloud is free and open source. We are currenty using Nextcloud for file sharing with our community and the general public.
+
+## Using Nextcloud
+
+Staff are issued a Nextcloud account when they join. Nextcloud publishes [extensive support documentation](https://docs.nextcloud.com/).
+
+## Nextcloud management
+
+Nextcloud (user) administration is managed by the Operations Coordinator. The Operations Coordinator as well as the Chief Executive have full adminstrative privileges. The Operations Coordinator is responsible for user and folder management, customization, making sure Nextcloud stays up-to-date and ensures proper on-boarding and off-boarding policies are in place and enforced.
+
+Tweaks in the Nextcloud configuration can be made after SSH'ing into the machine as root:
+
+`cipmas@nextcloud-publiccode:/var/www/nextcloud/nextcloud/config$ sudo nano config.php`
+
+* use `cd` to change directories
+* use `ls -l` to see what’s in the directory
+* use `grep -i` command to find specific things, such as trusted domains, which was needed to add sub domains making sure Nextcloud can be reached at e.g. 'files.publiccode.net' as well.
+
+## Privacy and GDPR
+
+Nextcloud has [an extensive GDPR complaince kit](https://nextcloud.com/gdpr/).
```

Comment:
This GDPR section doesn't explain what data is collected or how it's processed (as I asked on #815). We can bump this to live, but I'd like to see this corrected very soon.

Reply:
the short of it is that we self-host and thus need to adhere to our own internal GDPR and privacy policy. We can update [this page](https://about.publiccode.net/organization/privacy.html) with a section on Nextcloud if you think that's useful. But, since we are not actively using Nextcloud other than for sharing documents that can be downloaded by externals, it's probably not needed to enrich this page. There are no direct privacy implications for those who download the document / file. TL;DR, I can write that we run a self-hosted version of Nexcloud on a machine in Spain and then point to our general GDPR and privacy policy instead?

Reply:
We may actually need to point to Maadix's privacy policy, since Nextcloud is running on their hardware. I'll update the section accordingly.

Reply:
I updated this 3 days ago. It can be added to this release. https://github.com/publiccodenet/about/blob/ops-handbook-tools/activities/tool-management/nextcloud.md